batch/batch/front_end/front_end.py

```diff
@@ -1378,8 +1378,7 @@ async def parse_error(msg):
 LEFT JOIN billing_projects
   ON billing_projects.name = batches.billing_project
 WHERE `time_completed` >= %s AND
-  `time_completed` <= %s AND
-  billing_projects.`status` != 'deleted'
+  `time_completed` <= %s
```

Comment:
The billing_project index includes deleted status. I tested this just now on querying all of November, it takes 3.75 seconds without the deleted ones and 6.91 to with no status filter.

Billing project deletion is used by the tests to remove test billing projects. An open billing project doesn't have a delete button, just a close button, so I don't think it's likely to happen by accident.

```
mysql> explain SELECT   billing_project,   `user`,   CAST(SUM(IF(format_version < 3, batches.msec_mcpu, 0)) AS SIGNED) as msec_mcpu,   SUM(IF(format_version >= 3, `usage` * rate, NULL)) as cost FROM batches LEFT JOIN aggregated_batch_resources   ON aggregated_batch_resources.batch_id = batches.id LEFT JOIN resources
  ON resources.resource = aggregated_batch_resources.resource  LEFT JOIN billing_projects    ON billing_projects.name = batches.billing_project  WHERE `time_completed` >= 1606861283280 AND `time_completed` <= 1607638653000 and billing_projects.status != 'deleted' GROUP BY billing_project, `user`;
+----+-------------+----------------------------+------------+--------+------------------------------------------------------+------------------------+---------+-------------------------------------------+------+----------+--------------------------------------------------------+
| id | select_type | table                      | partitions | type   | possible_keys                                        | key                    | key_len | ref                                       | rows | filtered | Extra                                                  |
+----+-------------+----------------------------+------------+--------+------------------------------------------------------+------------------------+---------+-------------------------------------------+------+----------+--------------------------------------------------------+
|  1 | SIMPLE      | batches                    | NULL       | range  | batches_time_completed,batches_billing_project_state | batches_time_completed | 9       | NULL                                      | 5759 |   100.00 | Using index condition; Using temporary; Using filesort |
|  1 | SIMPLE      | billing_projects           | NULL       | eq_ref | PRIMARY,billing_project_status                       | PRIMARY                | 302     | batch.batches.billing_project             |    1 |    66.67 | Using where                                            |
|  1 | SIMPLE      | aggregated_batch_resources | NULL       | ref    | PRIMARY                                              | PRIMARY                | 8       | batch.batches.id                          |   36 |   100.00 | NULL                                                   |
|  1 | SIMPLE      | resources                  | NULL       | eq_ref | PRIMARY                                              | PRIMARY                | 302     | batch.aggregated_batch_resources.resource |    1 |   100.00 | NULL                                                   |
+----+-------------+----------------------------+------------+--------+------------------------------------------------------+------------------------+---------+-------------------------------------------+------+----------+--------------------------------------------------------+
```